openstack/cinder/templates/vcenter_datacenter_cinder_deployment.yaml

```diff
@@ -34,6 +36,8 @@ template: |
           application: cinder
           component: volume-vmware
           name: cinder-volume-vmware-{= name =}
+          vcenter: {= host =}
```

Comment:
Why don't we use `{= name =}` here? Looking at `name:` above, this should be the short-form `vc-a-1` or something like that, what we want, I would say.

Reply:
I would have preferred that too, but both the alerts defined for the `vcsa-exporter` and `vrops-exporter` have the fqdn as a prometheus label.

We can probably also take care with some prometheus relabelling actions to transform the Kubernetes label to fqdn in prometheus. Not sure, which of the two options is better.